### PR TITLE
Merge release back into main

### PR DESCRIPTION
**Pull request by joyceerhl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "1.56.0-insider"
+        "vscode": "1.56.0"
```
Comment (IanMatthewHuff) [inline package.json:27]: Can't fully remember our process on this engine number. Don't we keep it as -insider in main?

Comment (rchiodo) [reply package.json:27]: Yeah should be 1.57.0-insider

Comment (joyceerhl) [reply package.json:27]: I think we discussed this at bug bash and said we don't need to bump this to 1.57.0-insider right away, as that will block publishing nightly insiders builds until 1.57.0-insider becomes available

Comment (joyceerhl) [reply package.json:27]: But yes good point, closing this PR since I can't fix this directly in the release branch (which is the source of this PR)

**Pull request closed without merging**